### ISSUE TITLE
"Send" & "Send at" buttons don't send for HTML emails.

### ISSUE THREAD

**Issue by Geremia** [Extended-Thunder/send-later#60]
"Send" & "Send at" buttons don't send at all with 8.0.13rc2.

Console gives:
```
window is not defined sl3u.js:564
    preSendCheck jar:file:///home/Geremia/.thunderbird/ki2rin7bz.default/extensions/sendlater3@kamens.us.xpi!/experiments/sl3u.js:564
    preSendCheck self-hosted:850
    result resource://gre/modules/ExtensionParent.jsm:996
    withPendingBrowser resource://gre/modules/ExtensionParent.jsm:602
    result resource://gre/modules/ExtensionParent.jsm:996
    callAndLog resource://gre/modules/ExtensionParent.jsm:958
    recvAPICall resource://gre/modules/ExtensionParent.jsm:995
    InterpretGeneratorResume self-hosted:1158
    AsyncFunctionNext self-hosted:693
ERROR [SendLater]: Error: An unexpected error occurred static.js:26:13
    logger moz-extension://436da620-b5b3-4f7e-a303-c4d702d96a3f/utils/static.js:26
TypeError: browser is nullBrowserElementParent.jsm:81:21
    receiveMessage resource://gre/actors/BrowserElementParent.jsm:81
```

I seem to only have this problem with HTML emails; plaintext ones send.

**Comment by Geremia** [comment]
I get this same problem with 8.0.15rc2, too.

**Comment by jperryhouts** [comment]
This should be fixed by #68. Thanks for the debug log!

**Comment by Geremia** [comment]
@jperryhouts I fixed this issue in 8.0.15rc2 by making TB not prompt when it doubt if the email should be plaintext or HTML.

**Comment by jperryhouts** [comment]
Glad to hear it. Can you explain again what setting you changed? I'm not familiar with all of the different "prompt before sending if..." options in thunderbird.
